### ISSUE TITLE
8.4 redirects

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2740]
**What we're after** 
Making sure that the website users are getting the latest and most up to date content

**To do**
- [x]  Write www and proxy redirects for the following:
Annual No Fear Act Notice
subdirectory /info/presentations/  
Outdated internet brochure - redirect from before no longer working
Reporting contributions for debt retirement (old handout PDF)
FEC Glossary 2009 (old handout PDF)
Old testing the waters brochure 

- [x] Put in redirect in proxy and www for all old outreach files in directory list below to redirect to https://www.fec.gov/help-candidates-and-committees/trainings/: Moved from 8.3 #2699 
/info/conferences/
/info/conference_materials/
/info/roundtables/
/info/roundtable_materials/
/info/state_outreach/
/info/documents/
/info/downloads/

After the redirects are merged and deployed:
- [x] Test to ensure they are working.
- [x] Add above list to a ticket to take them down off of classic and transition

**Comment by kathycarothers** [comment]
There is an issue with 3 audio forms we need to workout before we can redirect these forms. They have been moved to 8.5.

**Comment by kathycarothers** [comment]
Redirects have been merged and tested. We have created a new ticket #2800 to take the old pages/folders/content down.
